frontend/assets/stylesheets/components/_bundle-header.scss

```diff
@@ -0,0 +1,31 @@
+
+/* Header */
+/* Green Part of the Header */
+.bundle-header--main-header {
+    background: #87be27;
+}
+
+/* White Part of the Header */
+.bundle-header--secondary-header {
+
+}
+
+.bundle-a__title {
+    @include text-title();
+    padding-top: 24px;
+}
+
+.bundle-a__tagline {
+    @include text-tag-line();
+    padding-bottom: 36px;
```

Comment:
We should try and use multiples of [`gs-baseline`](https://github.com/guardian/membership-frontend/blob/master/frontend/assets/stylesheets/_mixins.scss#L19) to represent vertical padding, to reduce the amount of magic numbers and help preserve a consistent vertical rhythm